consensus/src/dag/dag_network.rs

```diff
@@ -0,0 +1,21 @@
+use crate::dag::types::DAGMessage;
+use aptos_consensus_types::common::Author;
+use async_trait::async_trait;
+use std::time::Duration;
+
+pub trait RpcHandler {
+    type Ack;
+    type Message;
+
+    fn process(&mut self, message: Self::Message) -> anyhow::Result<Self::Ack>;
+}
+
+#[async_trait]
+pub trait DAGNetworkSender: Send + Sync {
+    async fn send_rpc(
+        &self,
+        receiver: Author,
+        message: DAGMessage,
```

Comment:
does it need to be DAGMessage? since we may want to move it to general consensus